### ISSUE TITLE
[Request] List of training set task names/map (similar to libero_suite_task_map)

### ISSUE THREAD

**Issue by yupeng-2003** [sylvestf/LIBERO-plus#32]
Hi ,

Thank you for the excellent work!

I was looking through the repository and noticed that `libero_suite_task_map` provides a clear mapping for the test set. I was wondering if you could share a similar list or mapping for the **training set**?

Thanks in advance for your help!

**Comment by steb6** [comment]
I am looking forward it as well
